tools/bouncer_worker/src/queues/modelQueueHandler.js

```diff
@@ -61,18 +61,20 @@ Handler.onMessageReceived = async (cmd, rid, callback) => {
 	const ridString = rid.toString();
 
 	try {
-		const { size } = fs.statSync(file);
-		const processInformation = Utils.gatherProcessInformation(
+		let processInformation = Utils.gatherProcessInformation(
 			user,
 			model,
 			database,
 			logLabel.label, // queue
-			file.split('.').pop().toString(), // filetype
-			size, // filesize
 			config.repoLicense,
 			ridString,
 		);
 
+		// Append process information with queue specific properties
+		const { size } = fs.statSync(file);
+		processInformation.FileType = file.split('.').pop().toString(), // filetype
```

Comment:
could we do this for drawing stoo, unless i've mistakened I don't think we're adding this info in the drawing queue handler

@Charence we're goig to have similar model processing documents for 2D Drawings, they can be DWGs which means file type can't distinguish them (as we also have DWGs)

What do you propose? do we add an additional field or do something like "DWG (2D)" as the file type?

Reply:
@carmenfan @sebjf I think it depends if DWG is the only case where it can store both 2D and 3D and we want to distinguish if it came from a 2D or 3D file.

1. If `DWG (2D)` is kind of a one-off special case and we assume the person looking at the records knows if a format is 2D or 3D/doesn't care if it is 2D or 3D then I guess this could work.
2. If the person looking at the records wants to easily separate 2D vs 3D ones and we don't assume they know what format is 2D or 3D then I think we could store an additional field.

Reply:
Hi @carmenfan , the reason its not done in the drawing queue is that bouncerWorker doesn't read the ref node, it only gets what's in the message in the queue (`database`,`model`,`user`,`revId`, for drawings).

Is the plan for the extension and size to be added to the rabbit mq message for drawings?

Reply:
ah right.

Ok let me just add it as part of the json since the backend has all that info

Reply:
@sebjf  will probably do it for both 3D and 2D if it makes a difference here

Reply:
Hi @carmenfan @Charence,

There is an argument to `gatherProcessInformation` that takes a queue name as a label for the logs:

https://github.com/3drepo/3drepobouncer/blob/7f001fd8c1b125f318ee73c9df46348d2d44d60e/tools/bouncer_worker/src/queues/modelQueueHandler.js#L70

If this is distinct to each handler, then it is effectively the processing type, if it can be used that way @Charence? (I have set it to `DRAWINGQ`).

Please let me know where to find the updated schema for the queue messages when its done and I'll add the additional members to the drawings handler.

Reply:
> ah right.
> 
> Ok let me just add it as part of the json since the backend has all that info

@sebjf  this will come in as `format` and `size` as part of the json file